src/components/publishing/sections/sections.tsx

```diff
@@ -34,10 +36,11 @@ const Sections: React.SFC<SectionsProps> = props => {
 function renderSections(article) {
   const sections = article.sections
   const renderedSections = sections.map((section, i) => {
-    const child = getSection(section)
+    const child = getSection(section, article.layout)
+    const layout = section.body ? findBlockquote(section.body, article.layout) : section.layout
```

Comment:
maybe would make more sense to set the layout on these text sections directly, so blockquote doesn't have to be identified on the client.